.github/workflows/dependency-review.yml

```
@@ -12,4 +12,4 @@ jobs:
       - name: Dependency Review
         uses: actions/dependency-review-action@3b139cfc5fae8b618d3eae3675e383bb1769c019 # v4.5.0
         with:
-          config-file: it-at-m/.github/workflow-configs/dependency_review.yaml@main
\ No newline at end of file
+          config-file: it-at-m/.github/workflow-configs/dependency-review.yml@main
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)